### PR TITLE
Read desi files

### PR DESCRIPTION
**Pull request by londumas**
@ngbusca, could you review this commit?
Please don't merge yet. We are testing the full pipeline.

### CODE REVIEW

**Comment by londumas** [comment]
works very well. It only requires to build by hand a fake drq from the desi pipeline.

File: py/picca/io.py [hunk 1]
```diff
@@ -156,17 +155,11 @@ def read_data(in_dir,drq,mode,zmin = 2.1,zmax = 3.5,nspec=None,log=None,keep_bal
...
-        typ = h[1]["TRUESPECTYPE"][:]
-        w = ["QSO" in t for t in typ]
-        w = sp.array(w)
-        ztable = {t:z for t,z in zip(tid[w],zs[w]) if z > zmin and z<zmax}
-        sys.stderr.write("Found {} qsos\n".format(len(ztable)))
-        return read_from_desi(nside,ztable,in_dir)
+        sys.stderr.write("Found {} qsos\n".format(len(zqso)))
+        return read_from_desi(nside,in_dir,thid,ra,dec,zqso,plate,mjd,fid,order)
+
+        return read_from_desi(nside,ztable,in_dir,order)
```
Comment (ngbusca) [inline py/picca/io.py]: it looks like this line is never reached!

Comment (londumas) [reply py/picca/io.py]: Could you show me an exemple of code that reproduces your bug ? It witks very well fir me

Comment (ngbusca) [reply py/picca/io.py]: it's just that l. 162 is never reached since there's a return statement in l. 160, no?

Comment (londumas) [reply py/picca/io.py]: indeed, sorry. Just corrected it.

**Pull request merged**